justega247/Maintenance-Tracker
# #160410605: Setup sass preprocessor

#### What does this PR do?
- This PR is for the setup of sass preprocessor to enable the use of .scss and .css files for styling the application.
#### Description of Task to be completed?
- Install node-sass and sass-loader packages from npm
- Setup sass-loader in webpack config to enable the transpiling of SCSS styles into  CSS
#### How should this be manually tested?
- From the root of the project, run `npm run dev-server` in your command line
- Make changes to the .settings.scss file in the base folder located inside the styles folder
- You should see your changes take effect in the browser
#### What are the relevant pivotal tracker stories?
- https://www.pivotaltracker.com/story/show/160410605
